baybe/_optional/chemistry.py

```diff
@@ -0,0 +1,23 @@
+"""Optional chemistry imports."""
```

Comment:
would it make sense to name the respecive files after the names of the optional dependency groups? eg `chem` here instead if `chemistry`